CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+## Thank You
+Thank you for showing interest in contributing to HDRmerge. It is people such as yourself who make this program and project possible.
+
+## Contributing as a Programmer
+- Announce and discuss your plans in GitHub before starting work.
+- Work in a new branch. Fork if necessary.
+- Keep branches small.
+- Use C++11.
+
+### Naming and Comments
+The naming isn't homogeneous throughout the code but here is a rough guideline:
+- *Identifiers* (variables, functions, methods, keys, enums, etc.) should be clear and unambiguous. Make them as long as necessary to ensure that your code is understandable to others and to your future self.
+- *Types* (classes, structs, enums, typedefs...) should be named with `UpperCamelCase`.
+- *Functions* and *methods* should be named with `lowerCamelCase`.
+- *Variables* should be named with `lower_underscores` to avoid conflicts.
+- *Enum values* should be named with `UPPER_UNDERSCORES`.
```

Comment:
Another item:
- Prefer templates and `constexpr` variables over macros. Avoid macros as much as possible (e.g. use `#pragma once` instead of traditional include guards).